ci/environment-regressions.yml

```diff
@@ -0,0 +1,8 @@
+channels:
```

Comment:
Maybe this is worth combining with the static site env? Especially if we want to start importing some of this functionality when generating summaries.